### DISCUSSION TITLE
问下IOS支持小程序webview控件捕获吗

### DISCUSSION THREAD

**Discussion by chenzf666** [SonicCloudOrg/sonic-server#446] [General]
IOS的webview控件捕获场景
1、safari的webview控件，能正常捕获
2、小程序的webview控件，拿不到handler，也无法捕获（原生控件能捕获）
问下小程序webview控件支持捕获吗？

**Comment by YeungHoiChiu** [comment]
微信需要开启xweb debug 

>  先在微信随便找个聊天，发送http://debugxweb.qq.com/?inspector=true    后点击打开
